client/src/components/Logout/__tests__/index.spec.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { create } from 'react-test-renderer';
+import { mount } from 'enzyme';
+import moxios from 'moxios';
+ 
+import Logout from './../../Logout';
+
+const mockComponent = props => {
+    return (
+        <Logout {...props} />
+    );
+};
+
+describe('<Logout />', () => {
+    it('should render component', () => {
+        const tree = create(mockComponent()).toJSON();
+        
+        expect(tree).toMatchSnapshot();
+    });
+
+    it('should redirect to home after some time', (done) => {
+        moxios.install();
+        moxios.stubRequest('/api/logout', {
+            status: 200,
+            response: {}
+        });
+
+        const props = {
+            history: {
+                push: jest.fn()
+            }
+        };
+        const instance = mount(mockComponent(props));
+
+        setTimeout(() => {
+            expect(props.history.push).toHaveBeenCalledWith('/');
```

Comment:
I do not like this expect! It does not work properly (at least it does not write a correct error when the condition changed to do not pass the test). I hope in the future I will find a better solution how to test (or implement) the Logout functionality.